### ISSUE TITLE
Unit 12 figures copyright

### ISSUE THREAD

**Issue by jamesthcheung** [MUTOR-2/ScienceOfMusic#11]
Are the two images in Unit 12 ok with copyright?

**Comment by maccallum** [comment]
That unit was done by Birgitta Burger and Clemens Wöllner, not Psyche. I reassigned to @fuxiao1208--Xiao, can you check with them about those images?

**Comment by fuxiao1208** [comment]
yes, they have, but i will ask them to write the copyright type to me

**Comment by fuxiao1208** [comment]
hi just asked Birgitta, she made the both images, and we could write her name on them :)
